### ISSUE TITLE
[Bug] `titlePrefix` in advanced story specifiers causes the story to crash with "Didn't find 'xyz' in CSF file"

### ISSUE THREAD

**Issue by JReinhold** [storybookjs/addon-svelte-csf#130]
### Describe the bug

Originally discussed at #120.

If a stories specifier uses [the advanced pattern with a title prefix](https://storybook.js.org/docs/react/configure/sidebar-and-urls#auto-title-prefixes) like this:

```
const config = {
  stories: [
    {
      directory: '../src',
      titlePrefix: 'Custom', // 👈 Configure the title prefix
    },
  ],
};

```

The story will fail to render with "Didn't find 'some-story-id' in CSF file", if it's been written using Svelte CSF.

This happens because the indexer in this addon ignores that title prefix when generating story IDs, but the Storybook CSF processing in the browser does _not_ ignore that title prefix, so the ID it is looking for will not match the IDs in the index.

### Steps to reproduce the behavior

Reproduction: https://github.com/mcmxcdev/storybookjs-addon-svelte-csf-120

### Screenshots and/or logs

![image](https://github.com/storybookjs/addon-svelte-csf/assets/16797721/237f6262-e45d-4cc9-b7c2-f0415eda1cef)

### Additional context

This bug can probably be fixed by using the `makeTitle` function that is passed in the indexer options (second parameter) to the indexer function. That function can be used to generate titles automatically, also taking into account any user title specified.

Using `makeTitle` to fix this bug will likely also solve #129 .

**Comment by j3rem1e** [comment]
Should be fixed with #134

**Comment by mcmxcdev** [comment]
> Should be fixed with #134

That would be amazing, I have already cloned the project and was thinking of fixing it myself! :tada:

**Comment by shilman** [comment]
:rocket: Issue was released in [`v4.0.2`](https://github.com/storybookjs/addon-svelte-csf/releases/tag/v4.0.2) :rocket:
